YSI_Game/y_weapondata/y_weapondata_entry.inc

```diff
@@ -566,3 +560,141 @@ stock Weapon_GetIDFromModelID(modelid)
 	}
 	return WEAPONDATA_INVALID_RETURN;
 }
+
+#define WEAPON_TYPE_UNKNOWN				WeaponType:0
+#define WEAPON_TYPE_MEELE 				WeaponType:1
+#define WEAPON_TYPE_PISTOL 				WeaponType:2
+#define WEAPON_TYPE_ASSAULTRIFLE 		WeaponType:3
+#define WEAPON_TYPE_SHOTGUN 			WeaponType:4
+#define WEAPON_TYPE_SUBMACHINEGUN 		WeaponType:5
+#define WEAPON_TYPE_RIFLE 				WeaponType:6
```

Comment:
Should the type numbers not match the slot numbers?  Or if not ohw are they different (please document the difference)?